packages/client/src/views/RegisterForm/RegisterForm.tsx

```diff
@@ -724,7 +724,7 @@ class RegisterFormView extends React.Component<FullProps, State> {
         data: {
           ...modifiedDeclaration.data,
           informant: {
-            informantType: modifiedDeclaration.data.informant.informantType
+            ...modifiedDeclaration.data.informant
```

Comment:
Why was this needed?

Reply:
the phone number field has a dynamicInitialValue but when the change is made, it is not persisted.

Reply:
I was looking at this and started wondering, isn't 

```ts
      modifiedDeclaration = {
        ...modifiedDeclaration,
        data: {
          ...modifiedDeclaration.data,
          informant: {
            ...modifiedDeclaration.data.informant
          }
        }
      }
```
the same as 

```ts
      modifiedDeclaration = {
        ...modifiedDeclaration,
        data: {
          ...modifiedDeclaration.data,
          informant: modifiedDeclaration.data.informant
        }
      }
```

which is the same as 

```ts
      modifiedDeclaration = {
        ...modifiedDeclaration,
        data: modifiedDeclaration.data
      }
```

which is the same as no changes at all right? 🤔